plane_mcp/tools/journeys.py

```
@@ -0,0 +1,314 @@
+        return match.group(1).upper(), int(match.group(2))
+
+    def get_issue_uuid(self, project_identifier: str, issue_sequence: int):
+        try:
+            item = self.client.work_items.retrieve_by_identifier(
+                workspace_slug=self.workspace_slug,
+                project_identifier=project_identifier,
+                issue_identifier=issue_sequence,
+                params=RetrieveQueryParams(expand="assignees,state")
+            )
+            return item.id, item
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)